src/manager.cpp

```diff
@@ -133,10 +133,12 @@ sdbusplus::async::task<> Manager::processPendingNotifications()
     for (const auto& path : fs::directory_iterator(NOTIFY_SERVICES_DIR))
     {
         _notifyReqs.emplace_back(std::make_unique<notify::NotifyService>(
-            _ctx, *_extDataIfaces, path, [this](notify::NotifyService* ptr) {
+            _ctx, *_extDataIfaces, path,
+            [this](notify::NotifyService* ptr) {
             std::erase_if(_notifyReqs,
                           [ptr](const auto& p) { return p.get() == ptr; });
-        }));
+        }, DEFAULT_RETRY_ATTEMPTS,
```

Comment:
Why are you passing here? 

You are using the static value so you could use in the constructor directly, right? or in the retry logic?